sdk/cosmosdb/arm-cosmosdb/samples-dev/tableResourcesUpdateTableThroughputSample.ts

```diff
@@ -21,7 +21,7 @@ dotenv.config();
  * This sample demonstrates how to Update RUs per second of an Azure Cosmos DB Table
  *
  * @summary Update RUs per second of an Azure Cosmos DB Table
- * x-ms-original-file: specification/cosmos-db/resource-manager/Microsoft.DocumentDB/preview/2023-03-15-preview/examples/CosmosDBTableThroughputUpdate.json
+ * x-ms-original-file: specification/cosmos-db/resource-manager/Microsoft.DocumentDB/stable/2023-09-15/examples/CosmosDBTableThroughputUpdate.json
```

Comment:
I do wonder how useful these samples are, I realize we're generating them based on what's in the specification folder, but they seem very niche and per-method rather than being scenario-focused.

Reply:
True. Let me create a separate issue to track them.

Reply:
I think those samples are published to the rest api docs site similar like this one https://learn.microsoft.com/en-us/rest/api/appservice/app-service-certificate-orders/get?tabs=JavaScript, we can not remove them.